packages/strapi-design-system/src/Json/JsonInput.js

```diff
@@ -0,0 +1,156 @@
+import React, { useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { jsonParseLinter, json } from '@codemirror/lang-json';
+import { useCodeMirror } from '@uiw/react-codemirror';
+
+import { Field, FieldLabel, FieldError, FieldHint } from '../Field';
+import { Stack } from '../Stack';
+import { JsonInputContainer } from './JsonInputContainer';
+import { markField, addMarks, filterMarks, lineHighlightMark } from './utils/decorationExtension';
+
+export const JsonInput = ({
```

Comment:
It is really just a question: should we call it `JsonInput` or `JSONInput`?

Reply:
`JsonInput` made more sense to me to stay consistent with the naming convention in the repo. What do you think, do you have a preference?

Reply:
I think I'd prefer "JSON", because it is an abbreviation. Also, [the spec uses uppercase letters only](https://datatracker.ietf.org/doc/html/rfc8259). @joshuaellis opinions?

Reply:
i'd make it `JSON` cause, it's `JSON` :)